src/index.js

```diff
@@ -91,7 +91,7 @@ const loadConfig = (requiredKeys, file, actionType) => {
     .then(res => res.json())
     .then(config => {
       requiredKeys.forEach(key => {
-        if (!config[key]) throw new Error(`${key} is not set in ${file}. Coral will not work correctly.`);
+        if (typeof config[key] === "undefined") throw new Error(`${key} is not set in ${file}. Coral will not work correctly.`);
```

Comment:
But if we just have 'falsey' checks then we can never have a config field with the value 0, false or "".  If someone writes a config file and puts in a null value that's on them.